docs/changelog.rst

```diff
@@ -8,6 +8,8 @@ Release Notes
 Bug fixes
 ~~~~~~~~~
 
+* Add support for Structural Pattern Matching
```

Comment:
Nitpick: Can we add a link to the PEP?

Reply:
We also need to add "(Reported by avelican in Issue 215 and contributed by Skurikhin Alexandr in PR 219.)"

Reply:
Oh, and this needs to be into a new section called "New Features" :) See 0.8.0 for an example.

Reply:
All done, thank you